manifests/params.pp

```diff
@@ -15,7 +15,7 @@
   $cli_try_sleep         = 10
 
   case $::osfamily {
-    'Debian': {
+    'Debian', 'Ubuntu': {
```

Comment:
I don't believe this is necesasary, on my Ubuntu 12.04 LTS the `osfamily` fact reports "Debian", not "Ubuntu"